tests/command_count_test.go

```diff
@@ -0,0 +1,41 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandCOUNT(t *testing.T) {
+	commandsCount := 40
+
+	subscriber := getLocalConnection()
```

Comment:
Why call this subscriber? This is just a simple local TCP connection, so name it `connection` at best.

Reply:
Sure